js/libs/keycloak-admin-client/src/defs/userRepresentation.ts

```diff
@@ -18,6 +18,7 @@ export default interface UserRepresentation {
 
   // optional from response
   attributes?: Record<string, any>;
+  unmanagedAttributes?: Record<string, any>;
```

Comment:
Looking at the code, this is never returned from the API, but added for a specific case in the Admin Console? If that is the case, this type should be added to the form fields in the Admin Console where it is used, and not to the Admin Client.

Reply:
I think we need this field here because it is the type of object managed by the user UI/forms. If you have a better idea to add this field only in the context of the admin console, perhaps it can work.

I remember you showing some code where we can extend an existing type with additional fields?

Reply:
Created a `UIUserRepresentation` type extending the original one.